tests/integration/test_deb_snap_migrate.py

```diff
@@ -0,0 +1,76 @@
+import pytest
```

Comment:
having multiple integration tests python modules will mean your constantly creating machines, and tearing them down, building new models over and over.  If that's the intent -- okay, but it's time consuming.  

Consider trying to get as much done in one model as possible